config.json

```
@@ -1825,8 +1825,8 @@
       "uuid": "23e1c40a-300c-4a44-bc32-f3440a057216"
     },
     {
-      "name": "Strings",
-      "slug": "strings",
+      "name": "String formatting",
```

Since this is the title that shows for example in the concept box in the tree, it should be "String Formatting".
```suggestion
      "name": "String Formatting",
```